gdrive_sync/api.py

```diff
@@ -485,3 +490,58 @@ def update_sync_status(website: Website, sync_datetime: datetime):
     website.sync_status = new_status
     website.sync_errors = (website.sync_errors or []) + errors
     website.save()
+
+
+@transaction.atomic
+def rename_file(obj_text_id, obj_new_filename):
```

Comment:
add a unit test?

Reply:
Added in 3fa3378092e68fa98a2403d76fd3fb79eeae9743.